index.html

```diff
@@ -1,15 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles/index.css" />
+    <link rel="stylesheet" href="styles/index.css">
 </head>
+
 <body>
+    <header class="header__header">
+        <h1>Weather Report</h1>
+        <span class="header__subtitle">
+            for the lovely city of 
+            <span id="headerCityName" class="header__city-name"></span>
```

Comment:
There is no default city value, so if someone lands on the page and tries to get the real temperature, they will get an error trying to retrieve the temp without clear messaging of what they have done wrong. 

We typically want a webpage to start in a state where the controls available are able to be interacted with so folks can see what the page can do and are not confused or think certain features may be broken. What would be an interesting default location for your page?